import-service/provider.tf

```diff
@@ -0,0 +1,7 @@
+provider "google" {
+  alias = "target"
+}
+
+provider "google-beta" {
+  alias = "target"
+}
```

Comment:
are these necessary? Are these here to support future changes in which we add more config to these providers?

Reply:
i was repeating incantations i don't understand. @gmalkov  ?